runtime/runtime.go

```diff
@@ -439,6 +439,7 @@ func (rt *Runtime) Serve(ctx context.Context) error {
 	signal.Notify(signalc, syscall.SIGINT, syscall.SIGTERM)
 
 	rt.serverInitMtx.Lock()
+	rt.Manager.ServerInitialized()
 	rt.serverInitialized = true
 	rt.serverInitMtx.Unlock()
```

Comment:
🤔 hmm it might be nitpicking, but perhaps we should move this below line 444? Just because it'll ensure that when the manager is informed that the server was initialized, the field will have been set already, too?